vllm/entrypoints/openai/serving_engine.py

```diff
@@ -98,9 +99,10 @@ def create_streaming_error_response(
         })
         return json_str
 
-    async def _check_model(
-        self, request: Union[CompletionRequest, ChatCompletionRequest,
-                             EmbeddingRequest]
+    def _check_model(
```

Comment:
Theoretically this change is not required, but if left as is, the endpoints of tokenize/detokenize should also be `async`. Since `async` is not required in this function, I decided to remove `async`, but the endpoints of tokenize/detokenize can be adapted instead.

Reply:
Aren't `tokenize` and `detokenize` already implemented as `async`?

Reply:
Sorry, I forgot about that. I thought it was `async`. Do I need to restore `async`, or just remove the remaining `async` from `tokenize`/`detokenize`?

Reply:
Since all the other endpoints are implemented as `async` and `_check_model` is used by all the other endpoints, I would prefer to keep these all as async for now

If there is a good reason to convert the rest of the endpoints away from `async`, we can consider it, but this should be done in a separate PR with clear logic for why